ios/FluentUI/Notification/FluentNotification.swift

```diff
@@ -329,6 +351,8 @@ public struct FluentNotification: View, ConfigurableTokenizedControl {
     @State private var innerContentsSize: CGSize = CGSize()
     @State private var attributedMessageSize: CGSize = CGSize()
     @State private var attributedTitleSize: CGSize = CGSize()
+    @available(iOS 15.0, *)
```

Comment:
FYI we can't have `@available` tags in the newest Xcode 14: https://github.com/microsoft/fluentui-apple/pull/1079